File: pinyougou/pinyougou-seckill-web/src/main/webapp/js/service/seckillGoodsService.js
app.service('seckillGoodsService',function ($http) {

    this.add=function (id) {
        return $http.get('/seckill/order/add.shtml?id='+id);
    }

    //根据ID到后台查询商品详情
    this.getOne=function (id) {
        return $http.get('/seckill/goods/one.shtml?id='+id);
    }

    //查询所有商品列表
    this.list=function () {
        return $http.get('/seckill/goods/list.shtml');

    }
})

File: pinyougou/pinyougou-seckill-interface/src/main/java/com/pinyougou/seckill/service/SeckillGoodsService.java
package com.pinyougou.seckill.service;

import com.pinyougou.model.SeckillGoods;

import java.util.List;

public interface SeckillGoodsService {

    /**
     * 根据商品的ID去Redis中查询
     * @param id
     * @return
     */
    SeckillGoods getOne(Long id);

    /**
     * 查询所有秒杀商品
     * @return
     */
    List<SeckillGoods> list();
}
